src/Features/OnboardingTasks/Task.php

```diff
@@ -380,24 +364,24 @@ public function get_json() {
 		$this->possibly_track_completion();
 
 		return array(
-			'id'             => $this->id,
-			'parentId'       => $this->parent_id,
-			'title'          => $this->title,
-			'canView'        => $this->can_view,
-			'content'        => $this->content,
-			'additionalInfo' => $this->additional_info,
-			'actionLabel'    => $this->action_label,
-			'actionUrl'      => $this->action_url,
-			'isComplete'     => $this->is_complete,
-			'time'           => $this->time,
-			'level'          => $this->level,
+			'id'             => $this->get_id(),
+			'parentId'       => $this->get_parent_id(),
+			'title'          => $this->get_title(),
+			'canView'        => $this->can_view(),
+			'content'        => $this->get_content(),
+			'additionalInfo' => $this->get_additional_info(),
+			'actionLabel'    => $this->get_action_label(),
+			'actionUrl'      => $this->get_action_url(),
+			'isComplete'     => $this->is_complete(),
+			'time'           => $this->get_time(),
+			'level'          => $this->get_level(),
 			'isActioned'     => $this->is_actioned(),
 			'isDismissed'    => $this->is_dismissed(),
-			'isDismissable'  => $this->is_dismissable,
+			'isDismissable'  => $this->is_dismissable(),
 			'isSnoozed'      => $this->is_snoozed(),
-			'isSnoozeable'   => $this->is_snoozeable,
-			'snoozedUntil'   => $this->snoozed_until,
-			'additionalData' => self::convert_object_to_camelcase( $this->additional_data ),
+			'isSnoozeable'   => $this->is_snoozeable(),
+			'snoozedUntil'   => $this->get_snoozed_until(),
+			'additionalData' => self::convert_object_to_camelcase( $this->get_additional_data() ),
 		);
 	}
 
```

Comment:
I am getting this PHP warning for the function below:
`PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-content/plugins/woocommerce-admin/src/Features/OnboardingTasks/Task.php on line 397`

Reply:
Since the default is `null` and I think it should be capable of mixed values, I updated this to only convert if we have an array of data.